pages/osx/dtrace.md

```diff
@@ -0,0 +1,38 @@
+# dtrace
+
+> Generic front-end to DTrace facility.
+> This command is a simple interface to invoke the D language compiler, retrieve buffered trace data from the DTrace kernel facility, and a set of basic routines to format and print traced data.
+> This requires root privileges.
+> More information: <https://www.unix.com/man-page/osx/1/dtrace/>.
+
+- Set dtrace's target data model which is the target architecture:
+
+`dtrace -arch {{arch_name}}`
+
+- Claim anonymous tracing state and display the traced data:
```

Comment:
```suggestion
- Claim [a]nonymous tracing state and display the traced data:
```

Reply:
Done, but why is this necessary?

Reply:
Mnemonics are used interchangeably to highlight long and short options in example descriptions.